.github/workflows/go.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: ^1.15
+        go-version: 1.15.15
```

Comment:
Let's pin to `1.15.*`, `1.15.x`, or `~1.15` (these should all be equivalent) to keep up-to-date with patches to the 1.15 minor version.